### PR TITLE
Add tests with txm

### PR DESCRIPTION
**Pull request by xqft**
Use [txm](https://github.com/anko/txm) for adding tests to every example. We could make it mandatory to write tests for making sure every example works as expected.

This PR only includes one test as an example, in another PR we should add the rest and fix examples that won't work if there's any.

### CODE REVIEW

**Comment by xqft** [comment]
100th pr :tada:

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        uses: actions/setup-node@v3
+        with:
+          node-version: latest
+
+      - name: Install txm
+        run: npm install -g txm
+
+      - name: Test with txm
+        run: | 
+          export CARGO_MANIFEST_DIR="/usr/local/Cellar/cairo-lang/v2.0.0-rc4/corelib"
```
Comment (MegaRedHand) [inline .github/workflows/ci.yml:31]: I think we should use `v2.1.0-rc1`. It adds a flag `--single-file`, needed when running from a single file. That way we don't need to update the test commands in each example later.

**Review by MegaRedHand — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (xqft) [reply .github/workflows/ci.yml:31]: Sorry I didn't quite catch that, could you elaborate? The commands to run need to be specified in every .md

Comment (MegaRedHand) [reply .github/workflows/ci.yml:31]: Nevermind. I didn't see we are using the install script. I was just saying that when we update to the latest compiler version, we'll need to add a `--single-file` flag to every `cairo-run` command.

**Review by rcatalan98 — COMMENTED** [review]
Great work, this will be extremely helpful. Could you fix the merge conflict? It looks like a simple conflict of one line.
Also I'll add as an issue the job of completing the missing tests for each file.

**Comment by xqft** [comment]
Done!

**Review by SantiagoPittella — APPROVED** [review]

**Pull request merged**